config/versions.ts

```diff
@@ -5,6 +5,8 @@ export const MASTODON_API_VERSION = '4.0.2'
 
 export const WILDEBEEST_VERSION = packagejson.version
 
-export function getVersion(): string {
-	return `${MASTODON_API_VERSION} (compatible; Wildebeest ${WILDEBEEST_VERSION})`
+export function getVersion(domain?: string): string {
+	return `Wildebeest/${WILDEBEEST_VERSION} (Mastodon/${MASTODON_API_VERSION} compatible; +https://${
+		domain ?? 'github.com/cloudflare/wildebeest'
+	})`
```

Comment:
Why is this change needed?  This looks like the User-Agent function:
```
export function getFederationUA(domain: string): string {
	return `Wildebeest/${WILDEBEEST_VERSION} (Mastodon/${MASTODON_API_VERSION}; +${domain})`
}
```

Reply:
You're right, it doesn't seem necessary. I *think* (but can't be certain) that when I wrote this code, this file contained the UA method, and the file that is now `./ua.ts` was named `./constants.`. In any case, this proposed change is meant to serve the same purpose as the `getFederationUA()` method, so I'll migrate my proposed changes to that file instead. Here's the rationale for the proposed change:

It's intended as a courtesy to Mastodon instance admins that expect the UA string for ActivityPub hosts to be structured in a specific way.* Speaking from my own experience, I use a combination of HTTP request headers that include the UA string as a custom version of Cloudflare's "Browser Integrity Check".

Here are some real-world examples:
`http.rb/5.1.1 (Mastodon/4.1.0; +https://mastodon.social/)`
`http.rb/5.1.0 (Mastodon/4.0.2; +https://mastodon.sdf.org/)`
`http.rb/5.1.1 (Mastodon/4.1.0+glitch; +https://ramblings.social/)`
`AodeRelay (ap-relay/v0.3.79-HEAD-4998cd3a; +https://relay.universeodon.com)`
`Akkoma 3.6.0-0-g45a11aa-develop; https://ti22.pro <admin@ti22.pro>`
`Pleroma 2.5.50-190-g4c0ab884.develop+soapbox; https://social.nettohikari.com <admin@nettohikari.com>`